supervision/draw/color.py

```diff
@@ -116,25 +122,40 @@ def as_bgr(self) -> Tuple[int, int, int]:
         """
         return self.b, self.g, self.r
 
+    @property
     @classmethod
-    def white(cls) -> Color:
-        return Color.from_hex(color_hex="#ffffff")
+    def WHITE(cls):
+        if cls._WHITE is None:
+            cls._WHITE = cls.from_hex("#ffffff")
+        return cls._WHITE
 
+    @property
     @classmethod
-    def black(cls) -> Color:
-        return Color.from_hex(color_hex="#000000")
+    def BLACK(cls):
+        if cls._BLACK is None:
+            cls._BLACK = cls.from_hex("#000000")
+        return cls._BLACK
 
+    @property
     @classmethod
-    def red(cls) -> Color:
```

Comment:
Please mark those methods as deprecated and announce dropping them in `supervision-0.20.0`.

Reply:
The same request goes for all other colors.